guide/english/laravel/index.md

```diff
@@ -3,7 +3,7 @@ title: Laravel
 ---
 # Laravel
 
-Laravel is a free and open-source PHP web framework available on [GitHub](https://github.com/laravel/laravel) and licensed under the terms of MIT License. It was created by Taylor Otwell who designed it with the objetive of enabling the development of web applications following the model–view–controller (MVC) architectural pattern and it is now the most starred PHP framework on GitHub. Some of the main features found in Laravel are different ways for accessing relational databases, utilities that aid in application deployment and maintenance and at last but not least modular packaging system with a dedicated dependency manager.
+Laravel is a free and open-source PHP web framework available on [GitHub](https://github.com/laravel/laravel) and licensed under the terms of MIT License. It was created by Taylor Otwell who designed it with the objective of enabling the development of web applications following the model–view–controller (MVC) architectural pattern and it is now the most starred PHP framework on GitHub. Some of the main features found in Laravel are different ways for accessing relational databases, utilities that aid in application deployment and maintenance and last but not least modular packaging system with a dedicated dependency manager.
```

Comment:
Let's split the last sentence into two: 
- Some of the main features found in Laravel are different ways for accessing relational databases, utilities that aid in application deployment and maintenance.
- And last but not least, modular packaging system with a dedicated dependency manager.